public/components/layer_control/layer_configuration_options.tsx

```diff
@@ -99,6 +113,28 @@ function LayerConfigurationOptions(props: MapsExlorerOptionsProps) {
         setValue={setLayerValue}
       />
 
+      <EuiSpacer size="s"/>
+
+      <EuiText size="xs">
+         <strong>
+         <FormattedMessage id="mapsExplorer.layerVisParams.zoomLevelLable" defaultMessage="Zoom Level*"/>
+           </strong>
+      </EuiText>
+
+      <EuiDualRange
+        min={0}
+        max={14}
+        step={1}
```

Comment:
can we define constants for these numbers?

Reply:
Yes, we cab